src/tools/update-fixtures.ts

```diff
@@ -37,13 +38,14 @@ async function updateFixture(options: UpdateFixtureOptions) {
       path.resolve(__dirname, '../../fixtures/packages/', options.folder);
   const sourceDir = path.join(fixturesDir, 'source');
   const convertedDir = path.join(fixturesDir, 'expected');
+  const branch = options.branch || 'master';
 
-  console.log(`Cloning ${options.repoUrl} to ${sourceDir}...`);
+  console.log(`Cloning ${options.repoUrl} #${branch} to ${sourceDir}...`);
   await mkdirp(fixturesDir);
   rimraf.sync(sourceDir);
 
   await exec(
-      `git clone ${options.repoUrl} ${sourceDir} --depth=1`,
+      `git clone ${options.repoUrl} ${sourceDir} --branch=${branch} --depth=1`,
```

Comment:
This looks like this requires the branch, but the branch arg is optional?

Reply:
it should be optional (so that we can use the default branch if none provided)

Reply:
Right above, it defaults to `master`. Just seemed simpler than modifying this command.

Reply:
ah right! Okay I'm happy with this, it doesn't support a different default branch, but that's a minor minor minor point.